packages/wordcount/src/stripConnectors.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Replaces items matched in the regex with spaces.
  *
- * @param {Object} settings The main settings object containing regular expressions
+ * @param {import("./index").WordCountSettings} settings The main settings object containing regular expressions
```

Comment:
Since those are used in only one place, opted in for using `import` directly, without a `typedef`.

Reply:
> Since those are used in only one place, opted in for using `import` directly, without a `typedef`.

Personally I feel it's fine to make these compromises where sensible.

Reply:
I obviously agree via making the compromise :) Resolving.